src/plugins/deps/geomap-lib.js

```diff
@@ -2422,10 +2431,18 @@ MapLayer.prototype.addToLegend = function() {
 	$label = $( "<label>", {
 		"for": "cb_" + this.id,
 		text: this.settings.title
-	} ).prepend( $chkBox );
+	} );
 
-	$li = $( "<li class='checkbox geomap-lgnd-layer'>" )
-		.append( $label, "<div id='sb_" + this.id + "'></div>" );
+	// Display large checkboxes if class is present
+	if ( $ul.closest( ".wb-geomap" ).hasClass( "large-checkboxes" ) ) {
```

Comment:
I think you can access directly to the element like `this.map.legendDiv` like you did later.


```suggestion
	if ( this.map.mapDiv.context.classList.contains( "large-checkboxes" ) ) {
```